src/store/facilityList/actions.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  FacilityList,
+  FacilityActionTypes,
+  UPDATE_FACILITYLIST,
+} from './types';
+
+export function updateFacilityList(list: FacilityList): FacilityActionTypes {
+  return {
+    type: UPDATE_FACILITYLIST,
+    payload: list,
+  } as const;
```

Comment:
nitpick but do you need `as const` here? I haven't seen this notation before so just clarifying.